src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -32,10 +32,10 @@
   --calcite-accordion-item-background: theme("backgroundColor.foreground.1");
 }
 
-:host-context([appearance="minimal"]) {
+:host([appearance="minimal"]) {
   --calcite-accordion-item-padding: var(--calcite-accordion-item-spacing-unit) 0;
 }
-:host-context([appearance="transparent"]) {
+:host([appearance="transparent"]) {
   --calcite-accordion-item-border: transparent;
```

Comment:
these CSS properties could be set on the parent item (accordion) instead of here.